sacred/experiment.py

```diff
@@ -62,7 +64,13 @@ def __init__(self, name: Optional[str] = None,
             Optional full path to the base directory of this experiment. This
             will set the scope for automatic source file discovery.
 
+        additional_gatherers : Dict[str, Callable], optional
```

Comment:
I would actually not repeat the type annotation in the docstrings as this creates redundancy.